src/web/components/elements/InteractiveBlockComponent.tsx

```diff
@@ -263,27 +270,36 @@ export const InteractiveBlockComponent = ({
 	}, [loaded]);
 
 	return (
-		<div
-			data-cypress={`interactive-element-${encodeURI(alt || '')}`}
-			ref={wrapperRef}
-			className={wrapperStyle(role, loaded)}
-		>
-			{!loaded && (
-				<Placeholder
-					height={decideHeight(role)}
-					shouldShimmer={false}
+		<>
+			<div
+				data-cypress={`interactive-element-${encodeURI(alt || '')}`}
+				ref={wrapperRef}
+				className={wrapperStyle(role, loaded)}
+			>
+				{!loaded && (
+					<Placeholder
+						height={decideHeight(role)}
+						shouldShimmer={false}
+					/>
+				)}
+				{!loaded && (
+					<a
+						ref={placeholderLinkRef}
+						data-name="placeholder"
+						className={placeholderLinkStyle}
+						href={url}
+					>
+						{alt}
+					</a>
+				)}
+			</div>
+			{caption && (
```

Comment:
This might be doing the right thing, not sure, but maybe the logic around `loaded` or not can be raised slightly so we `return`. That way we avoid the render falling through to here if we don't want it to?

Reply:
The `loaded` is referencing loading from external to React, not the loading of the component itself. We always want to render the caption if it exists, so I think this is correctly structured.

Reply:
Ah, gotcha 👍